TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/subsystems/WebcamAprilTags.java

```diff
@@ -0,0 +1,215 @@
+package org.firstinspires.ftc.teamcode.robot.subsystems;
+
+import android.annotation.SuppressLint;
+import android.util.Size;
+
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
+import org.firstinspires.ftc.teamcode.core.lib.autonomousControl.Pose2d;
+import org.firstinspires.ftc.teamcode.core.lib.gamepad.GamepadManager;
+import org.firstinspires.ftc.teamcode.core.lib.interfaces.Subsystem;
+import org.firstinspires.ftc.teamcode.robot.constants.AutonomousConstants;
+import org.firstinspires.ftc.vision.VisionPortal;
+import org.firstinspires.ftc.vision.apriltag.AprilTagDetection;
+import org.firstinspires.ftc.vision.apriltag.AprilTagLibrary;
+import org.firstinspires.ftc.vision.apriltag.AprilTagLibrary.Builder;
+import org.firstinspires.ftc.vision.apriltag.AprilTagProcessor;
+import org.openftc.easyopencv.OpenCvInternalCamera;
+
+import java.util.List;
+
+public class WebcamAprilTags implements Subsystem {
+    private static WebcamAprilTags instance;
+    public enum Sections{
+        BLUE_ALLIANCE_SIDE_3_NEXUSES,
+        BLUE_MIDDLE_3_NEXUSES,
+        RED_ALLIANCE_SIDE_3_NEXUSES,
+        RED_MIDDLE_3_NEXUSES,
+        BLUE_BACKWARDS_NEXUSES,
+        RED_BACKWARDS_NEXUSES,
+        BLUE_PLATAFORM_NEXUSES,
+        RED_PLATAFORM_NEXUSES,
+        UNKNOWN_SECTION
+    }
+    private Telemetry telemetry;
+    private static AprilTagProcessor aprilTag;
+    static double CAM_DIST_TO_CENTER = 0;
+    // distancia da camera até o centro do robo
+    static double CAM_ANGLE_TO_GROUND =0;
```

Comment:
add all constants to it's own class constants on the file constants

Reply:
Constants moved to AutonomousConstants file